aspnet/conceptual-overview/aspnet.rst

```diff
@@ -57,19 +77,21 @@ ASP.NET Core applications are defined using a public ``Startup`` class:
       }
   }
 
-The ``ConfigureServices`` method defines the services used by your application and the ``Configure`` method is used to define what middleware makes up your request pipeline. See :doc:`/fundamentals/startup` for more details.
+- :doc:`ConfigureServices </fundamentals/startup>`  defines the services used by your app (such as the ASP.NET MVC Core framework, Entity Framework Core, Identity, etc. )
+- :doc:`Configure </fundamentals/startup>` defines the :doc:`middleware </fundamentals/middleware>` in the request pipeline
```

Comment:
**Naming is hard.** It might be educational and comic relief to add something like, "It would be more consistent to call these `ConfigureServices` and `ConfigureMiddleware`.  Naming is hard."